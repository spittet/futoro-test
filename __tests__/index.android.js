/* eslint-env mocha */
/* eslint-disable */

import 'react-native';
import React from 'react';
import Index from '../index.android.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

jest.mock('react-native-camera', () => 'Camera');
jest.mock('react-native-fs', () => 'RNFS');

it('renders correctly', () => {
  const tree = renderer.create(
    <Index />
  );
});
