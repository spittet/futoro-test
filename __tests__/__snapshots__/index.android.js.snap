// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    style={
      Array [
        Object {
          "flex": 1,
          "flexDirection": "column-reverse",
        },
        Object {
          "backgroundColor": "#000",
        },
      ]
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        collapsable={undefined}
        pointerEvents="auto"
        style={
          Object {
            "backgroundColor": "#E9E9EF",
            "bottom": 0,
            "left": 0,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "shadowColor": "black",
            "shadowOffset": Object {
              "height": 0,
              "width": 0,
            },
            "shadowOpacity": 0.2,
            "shadowRadius": 5,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "flex": 1,
            }
          }
        >
          <Text
            accessible={true}
            allowFontScaling={true}
            disabled={false}
            ellipsizeMode="tail"
            style={
              Object {
                "fontSize": 20,
                "margin": 10,
                "textAlign": "center",
              }
            }
          >
            This is the main screen
          </Text>
          <View
            accessibilityComponentType="button"
            accessibilityLabel={undefined}
            accessibilityTraits={
              Array [
                "button",
              ]
            }
            accessible={true}
            collapsable={undefined}
            hitSlop={undefined}
            isTVSelectable={true}
            nativeID={undefined}
            onLayout={undefined}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
            testID={undefined}
            tvParallaxProperties={undefined}
          >
            <View
              style={
                Array [
                  Object {},
                ]
              }
            >
              <Text
                accessible={true}
                allowFontScaling={true}
                disabled={false}
                ellipsizeMode="tail"
                style={
                  Array [
                    Object {
                      "color": "#0C42FD",
                      "fontSize": 18,
                      "padding": 8,
                      "textAlign": "center",
                    },
                  ]
                }
              >
                Go to record screen
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      cardStyle={undefined}
      collapsable={undefined}
      getScreenDetails={[Function]}
      headerMode={undefined}
      index={0}
      layout={
        Object {
          "height": 0,
          "initHeight": 0,
          "initWidth": 0,
          "isMeasured": false,
          "width": 0,
        }
      }
      mode="float"
      navigation={
        Object {
          "dispatch": [Function],
          "goBack": [Function],
          "navigate": [Function],
          "setParams": [Function],
          "state": Object {
            "index": 0,
            "routes": Array [
              Object {
                "key": "Init-id-1500600905661-0",
                "routeName": "Main",
                "type": undefined,
              },
            ],
          },
        }
      }
      router={
        Object {
          "getActionForPathAndParams": [Function],
          "getComponentForRouteName": [Function],
          "getComponentForState": [Function],
          "getPathAndParamsForState": [Function],
          "getScreenConfig": [Function],
          "getScreenOptions": [Function],
          "getStateForAction": [Function],
        }
      }
      style={
        Object {
          "backgroundColor": "#F7F7F7",
          "borderBottomColor": "rgba(0, 0, 0, .3)",
          "borderBottomWidth": 0.5,
          "height": 64,
          "paddingTop": 20,
        }
      }
      transitionConfig={undefined}
    >
      <View
        style={
          Object {
            "flex": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "flexDirection": "row",
              },
            ]
          }
        >
          <View
            collapsable={undefined}
            pointerEvents="box-none"
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "transparent",
                "bottom": 0,
                "justifyContent": "center",
                "left": 70,
                "opacity": 1,
                "position": "absolute",
                "right": 70,
                "top": 0,
                "transform": Array [
                  Object {
                    "translateX": 0,
                  },
                ],
              }
            }
          >
            <Text
              accessibilityTraits="header"
              accessible={true}
              allowFontScaling={true}
              collapsable={undefined}
              disabled={false}
              ellipsizeMode="tail"
              numberOfLines={1}
              onLayout={[Function]}
              style={
                Object {
                  "color": "rgba(0, 0, 0, .9)",
                  "fontSize": 17,
                  "fontWeight": "600",
                  "marginHorizontal": 16,
                  "textAlign": "center",
                }
              }
            >
              Home Screen
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</View>
`;
